kong/plugins/path-prefix/handler.lua

```diff
@@ -1,8 +1,7 @@
-local plugin = require("kong.plugins.base_plugin"):extend()
-
-function plugin:new()
-    plugin.super.new(self, "path-prefix")
-end
+local plugin = {
+  PRIORITY = 800,
+  VERSION = "1.0.0"
```

Comment:
Should the rockspec version match this?
https://github.com/alexashley/kong-plugin-path-prefix/blob/master/kong-plugin-path-prefix-0.1.0-1.rockspec

Reply:
good call @clayharris - I suggest revving the version so the plugin can be used in both 2.x and 3.x versions.